### DISCUSSION TITLE
Web iptv-org.github.io what to do?

### DISCUSSION THREAD

**Discussion by Vinhphu120811** [iptv-org/community#307] [Q&A] [answered]
Web iptv-org.github.io What is it for, why is there a list of 30000 channels in the web, what is that list for?

**Comment by PopeyeTheSai10r** [comment, marked as answer]
iptv-org.github.io can be used to find out information about the project.
 
Some of the most common uses will be:
- Find out what channels are available
- Find out whether there's a steam available for certain channel
- Find out whether there's a guide available for certain channel
